common/pipelinegen/config_builder.go

```diff
@@ -266,3 +300,49 @@ func filterSmallBatchesProcessor(tracesProcessors []string) ([]string, bool) {
 
 	return filtered, smallBatchesEnabled
 }
+
+func AddServiceGraphScrapeConfig(c *config.Config) {
+	servicegraphScrape := config.GenericMap{
+		"job_name":        "servicegraph",
+		"scrape_interval": "10s",
+		"static_configs": []config.GenericMap{
+			{
+				"targets": []string{"127.0.0.1:9090"},
```

Comment:
We are using `127.0.0.1` , `0.0.0.0` and `localhost` interchangably across the code? Is there a need to standardize this? Use it from a const?